.github/workflows/integration.yml

```diff
@@ -23,13 +23,15 @@ jobs:
       with:
         fetch-depth: 0
 
-    - name: Update PR checklist for surge.sh as pending (pull request)
+    - name: Update PR checklist for surge.sh as queued (pull request)
```

Comment:
No need for this queued status. After the deployment is created, the state will go into pending. That state is enough.

Reply:
This step is still needed because we need to create the deployment itself first.